config/config.go

```diff
@@ -232,6 +237,8 @@ func (s *SourceSubnet) Validate() error {
 				return fmt.Errorf("invalid message contract address in EVM source subnet: %s", messageContractAddress)
 			}
 		}
+	case EVM_BLOCKHASH:
```

Comment:
It seems odd that Blockhash is considered its own VM type. Are we overloading the term "VM" here?